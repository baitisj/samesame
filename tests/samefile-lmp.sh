DIFF=diff
ECHO=echo
SED=sed

WORK=${srcdir}/work
LIST1=${WORK}/test1.lst
OUTPUT=${WORK}/test.out
CHECK=${srcdir}/tests/samefile-lmp.out
CMD=${srcdir}/samefile

(	${ECHO} 'samefile -A:'
	${CMD} -A < ${LIST1}
	${ECHO}
	${ECHO} 'samefile -Z:'
	${CMD} -Z < ${LIST1}
	${ECHO}
	${ECHO} 'samefile -L:'
	${CMD} -L < ${LIST1}
	${ECHO}

	${ECHO} 'samefile -irA:'
	${CMD} -irA < ${LIST1}
	${ECHO}
	${ECHO} 'samefile -irZ:'
	${CMD} -irZ < ${LIST1}
	${ECHO}
	${ECHO} 'samefile -irL:'
	${CMD} -irL < ${LIST1}
	${ECHO}

	${ECHO} 'samefile -irx:'
	${CMD} -irx < ${LIST1}
) | ${SED} -e 's/${srcdir}/./g' > ${OUTPUT}

${DIFF} -u ${CHECK} ${OUTPUT}
