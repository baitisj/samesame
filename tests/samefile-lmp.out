samefile -A:
11	./work/test1/file1a	./work/test1/file2a	=	2	3
11	./work/test1/file1a	./work/test1/file3a	=	2	1

samefile -Z:
11	./work/test1/file3a	./work/test1/file2a	=	1	3
11	./work/test1/file3a	./work/test1/file1a	=	1	2

samefile -L:
11	./work/test1/file2a	./work/test1/file1a	=	3	2
11	./work/test1/file2a	./work/test1/file3a	=	3	1

samefile -irA:
11	./work/test1/file1a	./work/test1/file1b	[2]
11	./work/test1/file2a	./work/test1/file2b	[3]
11	./work/test1/file2a	./work/test1/file2c	[3]
11	./work/test1/file1a	./work/test1/file2a	=	2	3
11	./work/test1/file1a	./work/test1/file2b	=	2	3
11	./work/test1/file1a	./work/test1/file2c	=	2	3
11	./work/test1/file1a	./work/test1/file3a	=	2	1

samefile -irZ:
11	./work/test1/file2c	./work/test1/file2b	[3]
11	./work/test1/file2c	./work/test1/file2a	[3]
11	./work/test1/file1b	./work/test1/file1a	[2]
11	./work/test1/file3a	./work/test1/file2c	=	1	3
11	./work/test1/file3a	./work/test1/file2b	=	1	3
11	./work/test1/file3a	./work/test1/file2a	=	1	3
11	./work/test1/file3a	./work/test1/file1b	=	1	2
11	./work/test1/file3a	./work/test1/file1a	=	1	2

samefile -irL:
11	./work/test1/file2a	./work/test1/file2b	[3]
11	./work/test1/file2a	./work/test1/file2c	[3]
11	./work/test1/file1a	./work/test1/file1b	[2]
11	./work/test1/file2a	./work/test1/file1a	=	3	2
11	./work/test1/file2a	./work/test1/file1b	=	3	2
11	./work/test1/file2a	./work/test1/file3a	=	3	1

samefile -irx:
11	./work/test1/file1a	./work/test1/file1b	[2]
11	./work/test1/file2a	./work/test1/file2b	[3]
11	./work/test1/file2a	./work/test1/file2c	[3]
11	./work/test1/file2b	./work/test1/file2c	[3]
11	./work/test1/file1a	./work/test1/file2a	=	2	3
11	./work/test1/file1a	./work/test1/file2b	=	2	3
11	./work/test1/file1a	./work/test1/file2c	=	2	3
11	./work/test1/file1b	./work/test1/file2a	=	2	3
11	./work/test1/file1b	./work/test1/file2b	=	2	3
11	./work/test1/file1b	./work/test1/file2c	=	2	3
11	./work/test1/file1a	./work/test1/file3a	=	2	1
11	./work/test1/file1b	./work/test1/file3a	=	2	1
11	./work/test1/file2a	./work/test1/file3a	=	3	1
11	./work/test1/file2b	./work/test1/file3a	=	3	1
11	./work/test1/file2c	./work/test1/file3a	=	3	1
